ui-participant/src/hub/consent/ConsentView.tsx

```diff
@@ -78,6 +78,10 @@ function RawConsentView({ form, enrollee, resumableData, pager, studyShortcode,
           type: 'SUCCESS'
         }
       }
+
+      if (!response.enrollee.profile) {
+        response.enrollee.profile = enrollee.profile
+      }
       updateEnrollee(response.enrollee).then(() => {
```

Comment:
quick fix: this updateEnrollee call during consent submission would strip the profile off the enrollee